src/main.js

```diff
@@ -110,7 +114,17 @@ class Entry {
  * @returns {Array} An array containing entry objects.
  */
 const parseVarFromType = envSpecAsArray => {
-  return (envSpecAsArrayParsed = envSpecAsArray.map(element => {
+  //in case a line started with "#" we would have a blank environmental variable
+  envSpecAsArray = envSpecAsArray.filter(line => line[0] !== "#");
+
+  return envSpecAsArray.map(element => {
+    comment = "";
```

Comment:
Instead of initialising this with an empty value and using if statements below, we can use regular expressions instead.

Something like that could make the cut:

```javascript
const commentRegexp = /#(.+)$/;
const commentMatch = element.match(commentRegexp);
const comment = (commentMatch) ? commentMatch[1] : null;
```

Or something like that 😄